packages/generator-langium/templates/cli/src/cli/cli-util.ts

```diff
@@ -1,10 +1,11 @@
-import type { AstNode, LangiumDocument, LangiumServices } from 'langium';
+import type { AstNode, LangiumCoreServices, LangiumDocument } from 'langium';
+import type { LangiumServices } from 'langium/lsp';
```

Comment:
Hmm probably not. Although it's used in the generated `extractAstNode` function below, it's just passed to `extractDocument`, which expects a core service set rather than an lsp-enriched on. Dropping it and updating the one usage should work just fine.